devel/dbus-python/Portfile

```diff
@@ -3,9 +3,9 @@
 PortSystem      1.0
 
 name            dbus-python
-version         1.2.18
+version         1.3.2
 
-set python_versions {27 35 36 37 38 39 310 311}
+set python_versions {27 35 36 37 38 39 310 311 312}
 
 # this default version should stay synchronized with python_get_default_version
 #    in the python PortGroup
```

Comment:
please set `python_default_version` below to 311 as that is the current default in MacPorts

Reply:
Done